sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/KeyVaultClient.java

```diff
@@ -320,6 +320,9 @@ public Certificate[] getCertificateChain(String alias) {
         headers.put("Authorization", "Bearer " + getAccessToken());
         String uri = keyVaultUri + "secrets/" + alias + API_VERSION_POSTFIX;
         String response = HttpUtil.get(uri, headers);
+        if (response == null) {
+            throw new NullPointerException();
+        }
```

Comment:
What's the difference here?

Reply:
To provide the stack trace when it is null.